test/testhelper.h

```diff
@@ -17,7 +17,7 @@ class HttpCredentialsTest : public HttpCredentials {
     }
 };
 
-static FolderDefinition folderDefinition(const QString &path) {
+static inline FolderDefinition folderDefinition(const QString &path) {
```

Comment:
Well this is a false positive since it's used... I guess this is while building the remote wipe test? This one is currently commented out so that'd explain.

In that case I'd argue the right fix is reenabling the test and making sure it passes, not to rub the warning under the carpet like that.

Reply:
The function is used in `test/testfolderman.cpp`, but as the body is defined in the header file, it is an *inline* function.

Reply:
Well the warning says defined but unused. That's really why I think you get the warning during compilation of test/testremotewipe.cpp and not test/testfolderman.cpp. Otherwise there'd be something really wrong with a compiler issuing unused function warnings for a used function...

Reply:
I've been looking into the remotewipe test a bit and I would lean toward just removing it from the build.

Could you try this instead of adding inline there? I'd expect the warning to be gone.